### PR TITLE
Added Matt's landing updates

### PR DESCRIPTION
**Pull request by GregTrifan**

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ sidebar_position: 1
 
 # Short Intro to ColourBox
 
-Let's discover **ColourBox in less than 5 minutes**.
+Let's  abc d discover **ColourBox in less than 5 minutes**.
```
Comment (deeNewearth) [inline docs/intro.md]: Please remove this change. I think it was a test change. We don't need to commit this

File: docs/test.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+---
```
Comment (deeNewearth) [inline docs/test.md]: Can we give this file a better file name then test.md

**Review by deeNewearth — CHANGES_REQUESTED** [review]
left comments

File: docs/test.md (hunk 2 above)
Comment (GregTrifan) [reply docs/test.md]: renamed it to `Colourbox-litepaper.md`

**Review by deeNewearth — APPROVED** [review]

**Pull request merged**